draft-ietf-tls-tls13.md

```diff
@@ -5278,18 +5278,24 @@ The registries and their allocation policies are below:
 
 - TLS Alert Registry: Future values are allocated via Standards
   Action {{RFC8126}}. IANA \[SHALL update/has updated] this registry
-  to include values for "missing_extension" and "certificate_required".
+  to include values for "missing_extension" and "certificate_required".  The
+  "DTLS-OK" column is marked as "Yes" for each new alert.
 
 - TLS HandshakeType Registry: Future values are allocated via
   Standards Action {{RFC8126}}. IANA \[SHALL update/has updated] this registry
   to rename item 4 from "NewSessionTicket" to "new_session_ticket"
   and to add the "hello_retry_request_RESERVED", "encrypted_extensions",
-  "end_of_early_data", "key_update", and "message_hash" values.
+  "end_of_early_data", "key_update", and "message_hash" values.  The "DTLS-OK"
+  are marked as "Yes" for each of these additions.
```

Comment:
I guess we were maybe talking about removing key_update for DTLS in favor of just using the epoch, but presumably DLTS 1.3 could update the registry as needed.
Though, hmm, since DTLS 1.3 doesn't actually exist yet and these are 1.3-only messages, maybe it makes sense to not mark them okay until DTLS 1.3 exists, i.e., in the DTLS 1.3 IANA considerations.

Reply:
The current plan is to use key_update in DTLS.  There are a few reasons why maintaining some degree of parity with TLS is nice here.  Asymmetric updates are one.